### PR TITLE
Refactor tests pa

### PR DESCRIPTION
**Pull request by panosatha**
- Small updates in the benefits code to avoid repetative code and add documentation
- Refactoring of tests for benefits by implementing unit tests for each method

### CODE REVIEW

File: flood_adapt/object_model/benefit.py [hunk 1]
```diff
@@ -498,29 +548,60 @@ def _make_html(self, cba):
...
+        Returns
+        -------
+        IBenefit
+            a Benefit object
+        """
 
         obj = Benefit()
         with open(filepath, mode="rb") as fp:
             toml = tomli.load(fp)
         obj.attrs = BenefitModel.parse_obj(toml)
```
Comment (Santonia27) [inline flood_adapt/object_model/benefit.py:568]: parse_object throws an error in the file on my local maschine. Not sure if there's something missing

File: tests/test_object_model/test_benefit.py
Comment (Santonia27) [inline tests/test_object_model/test_benefit.py:1]: Should there be a small test for "def save(self, filepath: Union[str, os.PathLike]): ?

**Review by Santonia27 — APPROVED** [review]
just had minor comments/questions

File: tests/test_object_model/test_benefit.py
Comment (panosatha) [reply tests/test_object_model/test_benefit.py:1]: nice catch!

**Review by dladrichem — APPROVED** [review]

**Pull request merged**